tasks/gru_nose.py

```diff
@@ -18,9 +19,8 @@ class GruNosePoster(BaseTask):
         while True:
             now = datetime.datetime.now()
             if now.hour == 20:
-                print("It's 8:00pm!")  # TODO add to log
                 try:
-                    await self.channel.send(file=discord.File(os.path.join(BASE_PATH, 'gru/{}.png'.format(now.date()))))
-                except FileNotFoundError:
-                    pass  # TODO log failure
+                    await self.channel.send(file=discord.File(os.path.join(BASE_PATH, 'ygru/{}.png'.format(now.date()))))
```

Comment:
Is this correct? Is the directory path supposed to be `ygru`?

Reply:
nope, good catch